packages/data-models/src/homonym.js

```diff
@@ -95,32 +103,29 @@ class Homonym {
   }
 
   /**
-   * Returns a language code of a homonym (ISO 639-3).
+   * Returns a language of the homonym..
    * Homonym does not have a language property, only lemmas and inflections do. We assume that all lemmas
    * and inflections within the same homonym will have the same language, and we can determine a language
-   * by using language property of the first lemma. We chan change this logic in the future if we'll need to.
+   * by using language property of the first lemma. We can change this logic in the future if we'll need to.
    *
-   * @returns {string} A language code, as defined in the `languages` object.
+   * @returns {Language} A language of the homonym.
    */
   get language () {
-    Logger.getInstance().warn('Please use languageID instead')
-    return LMF.getLanguageCodeFromId(this.languageID)
+    if (this.lexemes && this.lexemes[0] && this.lexemes[0].lemma && this.lexemes[0].lemma.language) {
```

Comment:
May be it is worth to create a specific check method inside Lexeme class?

Reply:
I think it's a very good proposal, I will create a method for the check.